test/shortener_test.js

```diff
@@ -16,18 +16,31 @@ exports['urlshorener goo.gle'] = function (test) {
 };
 
 exports['urlshorener bit.ly'] = function (test) {
-  var url = 'http://bit.ly';
-  var key = 'KEY';
-  var format = 'json';
-	var s = new shortener.Shortener();
-	s.bitly(url, key, format, function(result) {
-		test.deepEqual(result
-                   , {
-                   	shortUrl: 'http://bit.ly/wq5Qeg'
-                   	, longUrl: 'http://bit.ly/'
-                   }
-                   , "Deep equal for bit.ly");
+  	var url = 'http://bit.ly';
+  	var key = 'KEY';
+  	var format = 'json';
+  	var s = new shortener.Shortener();
+  	s.bitly(url, key, format, function(result) {
+  		test.deepEqual(result
+					   , {
+						   shortUrl: 'http://bit.ly/wq5Qeg'
+						   , longUrl: 'http://bit.ly/'
+					   }
+					   , "Deep equal for bit.ly");
     test.done();
 	});
 };
 
+exports['urlshorener readability'] = function (test) {
+  	var url = 'https://www.readability.com/';
+  	var s = new shortener.Shortener();
+  	s.bitly(url, function(result) {
```

Comment:
This is a shame for me. Fixed.
Also, can I know how do you run tests? Because I'm just checking with `sample.js` file.